content/en/synthetics/multistep.md

```diff
@@ -91,18 +93,11 @@ By default, a maximum of 10 steps can be created. To increase this limit, reach
 
    {{% tab "Request Body" %}}
 
-   * **Body type**: Select the type of the request body (`text/plain`, `application/json`, `text/xml`, `text/html`, `application/x-www-form-urlencoded`, or `None`) you want to add to your HTTP request.
+   * **Body type**: Select the type of the request body (`text/plain`, `application/json`, `text/xml`, `text/html`, `application/x-www-form-urlencoded`, `GraphQL`, or `None`) you want to add to your HTTP request.
```

Comment:
nice catch !

Reply:
can you add it the the 
- HTTP tests page
- HTTP requests for browser tests in /browser_tests/actions/ page
too please